internal/command/deploy/deploy.go

```diff
@@ -248,6 +249,9 @@ func run(ctx context.Context) error {
 }
 
 func DeployWithConfig(ctx context.Context, appConfig *appconfig.Config, forceYes bool) (err error) {
+	ctx, span := tracing.GetTracer().Start(ctx, "deploy_with_config")
+	defer span.End()
```

Comment:
```suggestion

```
You don't need to create a new span. you can just pull the existing span from context and add your stuff on it.
https://pkg.go.dev/go.opentelemetry.io/otel/trace#SpanFromContext